app/views/fragments/javascriptFirstSteps.scala.html

```diff
@@ -29,13 +29,14 @@
         channel: 'Subscriber',
         slug: null,
         name: document.title,
-        publication: 'GU.co.uk',
-        commissioningDesk: 'Subscriber',
         product: 'Subscriber',
-        edition: 'UK',
         productData: null
     };
 
+    guardian.trackerUrl = '@{Config.trackerUrl}';
+    guardian.ignorePageLoadTracking = @{jsVars.ignorePageLoadTracking};
```

Comment:
I *think* this might already get merged here: https://github.com/guardian/subscriptions-frontend/blob/event-tracking/app/views/fragments/javascriptFirstSteps.scala.html#L16 so you might be overriding it here?

(I don't love having this JsVars stringify stuff at the root `guardian`, if it's dynamic we should move it down a level to avoid conflicts. But that's a problem for another day)